libs/community/langchain_community/vectorstores/bigquery_vector_search.py

```diff
@@ -192,6 +193,11 @@ def _initialize_vector_index(self) -> Any:
         if self._have_index or self._creating_index:
             # Already have an index or in the process of creating one.
             return
+        table = self.bq_client.get_table(self.vectors_table)
+        if (table.num_rows or 0) < _MIN_INDEX_ROWS:
+            # Not enough rows to create index.
+            self._logger.debug("Not enough rows to create a vector index.")
+            return
```

Comment:
should we raise an Exception maybe?

Reply:
We don't need to raise an exception here, since: 
1. We'll only create the VECTOR INDEX if the row is > `_MIN_INDEX_ROWS`
2. We don't use the index when the number or rows is small, but vector search still returns correct results (without using the index) -> brute force results.

Reply:
I see, thx.